### PR TITLE
Issue 1001: Missing tooltips / hovered information

### LINKED ISSUE #1001: Missing tooltips / hovered information
The following Equipments / Items are missing tooltips information.  (not sure the right term: the text that is displayed when mouse hover).

I find it really helps to understand the aircraft.  Also, it helps when in low light condition (ex: evening, night flying).

# KMA-20 (Audio panel)
* Marker selection

# KX-165 (NAV/COM)
* Swap button
* Frequency knob
* NAV: Pull/Ident: display both volume (already done) and Ident status (missing).

# KR-87 (ADF)
* All buttons & knob

# KI-266 (DME)
* All buttons & knob

# KAP-140 (Autopilot)
* Edit knob

For reference:

image

### PR DESCRIPTION
**Pull request by mathieugouin**
Fixes #1001 

Remaining issue: dual input data for the same tooltip is partially working (NAV1&2 volume knob).  The volume is always refreshed, but not the push/pull state.

### CODE REVIEW

File: Models/Interior/Panel/Instruments/kma20/kma20.xml [hunk 1]
```diff
@@ -602,7 +622,7 @@
                 <label>Microphone: %s</label>
                 <property alias="../../../../params/knob"/>
                 <mapping>nasal</mapping>
-                <script>(arg[0] == -1) ? "COM1" : (arg[0]==1) ? "External" : "COM2"</script>
+                <script>return (arg[0] == -1) ? "COM1" : (arg[0]==1) ? "External" : "COM2";</script>
```
Comment (onox) [inline Models/Interior/Panel/Instruments/kma20/kma20.xml]: Could you change this nested ternary operators to the way you do in `dme.xml`? A single ternary operator is fine, but I find nested ones not very readable. Also make sure you put a space around operators (`==`)

File: Models/Interior/Panel/Instruments/kma20/kma20.xml [hunk 2]
```diff
@@ -566,6 +576,16 @@
...
             </binding>
         </action>
+        <hovered>
+            <binding>
+                <command>set-tooltip</command>
+                <tooltip-id>kma20-sens</tooltip-id>
+                <label>Marker Beacon Sensitivity: %s</label>
+                <property alias="../../../../params/sens"/>
+                <mapping>nasal</mapping>
+                <script>return (arg[0] == -1) ? "HIGH" : (arg[0] == 0) ? "LOW" : "TEST";</script>
```
Comment (onox) [inline Models/Interior/Panel/Instruments/kma20/kma20.xml]: See comment below.

File: Models/Interior/Panel/Instruments/kma20/kma20.xml [hunk 3]
```diff
@@ -505,6 +505,16 @@
...
```
Comment (onox) [inline Models/Interior/Panel/Instruments/kma20/kma20.xml]: See comment below.

**Comment by onox** [comment]
@mathieugouin When you have added the 9th commit (with the requested changes from the review), can you squash the commits as follows:

```sh
git reset --soft HEAD~9
git commit -m "Add tooltips for DME, radios, KMA-20, ADF, and AP"
```

Then check that everything is ok and force push.

**Comment by mathieugouin** [comment]
Implemented changes requested.

**Comment by onox** [comment]
Thanks for the contribution @mathieugouin :+1: :smile:

**Pull request merged**